src/api/comments.ts

```diff
@@ -0,0 +1,20 @@
+import { client } from '../utils/fetchClient';
+import { Comment } from '../types/Comment';
+
+export const getComments = async (postId: number | undefined) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
```

Comment:
so, if the function takes undefined, it means that we would have a query
/comments?postId=undefined
Right?)
handle it in the upper component)
```suggestion
export const getComments = async (postId: number) => {
  return client.get<Comment[]>(`/comments?postId=${postId}`);
};
```